packages/lib-forge/templates/script-template.py

```
@@ -0,0 +1,42 @@
+        Retorna lista vazia se não encontrar resultados.
+
+    Lança:
+        Exception: Quando a operação falha, com mensagem explicativa em português
+    """
+    try:
+        # Implementação aqui
+        resultado = []
+
+        return resultado
+
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)